conf/task_defaults/oscd_all.yaml

```diff
@@ -18,4 +18,8 @@ experiment:
     num_workers: 0
     val_split_pct: 0.5
     bands: "all"
+    pad_size:
+      - 64
+      - 64
     num_patches_per_tile: 1
+
```

Comment:
Can you remove the empty line?

Reply:
didn't you just ask to add an empty line?

Reply:
EOL char and newline char are two different things.

Reply:
Which chars are those? I'm only familiar with "\n" or "\r\n". (It looks like this file ends with two newlines though)

Reply:
Okay, I guess you're right, previously there was no "\n", now there's "\n\n", the goal is "\n". My text editor adds this automatically. I think most editors can be configured to do this.

Reply:
kk np, just making sure I understand (char codes are wayyyy in the past for me)